docs/modules/ROOT/pages/relay.adoc

```diff
@@ -160,6 +160,228 @@ All private keys are stored in the AWS Key Management Service. Keys are generate
+
+curl \
+  -X POST \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H "X-Api-Key: $KEY" \
+  -H "Authorization: Bearer $TOKEN" \
+  -d "$DATA" \
+    "$API_URL/txs"
+```
+
+You would receive a response in the following format:
+
+```TypeScript
+export type Address = string;
+export type BigUInt = string | number;
+export type Hex = string;
+export type Speed = 'safeLow' | 'average' | 'fast' | 'fastest';
+export type Status = 'pending' | 'sent' | 'submitted' | 'inmempool' | 'mined' | 'confirmed';
+
+export interface TransactionResponse {
+  transactionId: string; // Defender tx id
+  hash: string; // Ethereum hash
+  to: Address;
+  from: Address;
+  value: string;
+  data: string;
+  speed: Speed;
+  gasPrice: number;
+  gasLimit: number;
+  nonce: number;
+  status: Status;
+  chainId: number;
+}
+```
+==== Query Transaction
+To retrive a transaction status and data make a GET request to the `txs` endpoint with the Defender transaction id.
+
+An example of the request:
+
+```bash
+
+curl \
+  -X GET \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H "X-Api-Key: $KEY" \
+  -H "Authorization: Bearer $TOKEN" \
+    "$API_URL/txs/$ID"
+
+```
+
+You would receive a `TransactionResponse` object, e.g.:
+
+```JSON
+{
+   "chainId":4,
+   "hash":"0xcef95469a9f02757f0968ec8c11449ae5e7486073075381dcd62bacec9e5d627",
+   "transactionId":"affba150-e563-441e-ae49-04bd6050979a",
+   "value":"0x1",
+   "gasPrice":1000000000,
+   "gasLimit":21000,
+   "to":"0x179810822f56b0e79469189741a3fa5f2f9a7631",
+   "from":"0xbce0b5b71668e42d908e387b68dba91789c932b8",
+   "data":"0x",
+   "nonce":160,
+   "status":"mined",
+   "speed":"fast"
+}
+```
+
+=== Sign Endpoint
+To sign an arbitrary data with Relay API make a POST request with a payload contain hex string. Payload format:
+
+```TypeScript
+export interface SignMessagePayload {
+  message: Hex;
+}
+```
+
+An example of the request:
+
+```bash
+DATA='{ "message": "0x0123456789abcdef" }'
+
+curl \
+  -X POST \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H "X-Api-Key: $KEY" \
+  -H "Authorization: Bearer $TOKEN" \
+  -d "$DATA" \
+    "$API_URL/sign"
+```
+You would receive a response in the following format:
+
+```TypeScript
+export interface SignedMessagePayload {
+  sig: Hex;
+  r: Hex;
+  s: Hex;
+  v: number;
+}
+```
+
+An example of the response:
+
+```JSON
+{
+   "r":"0x819b2645a0b73494724dac355e6ecfc983d94597b533d34fe3ecd0277046a1eb",
+   "s":"0x3b73c695b47dd275d17246d86bbfe35f112a7bdb5bf4a5a1a8e22fe37dfd005a",
+   "v":44,
+   "sig":"0x819b2645a0b73494724dac355e6ecfc983d94597b533d34fe3ecd0277046a1eb3b73c695b47dd275d17246d86bbfe35f112a7bdb5bf4a5a1a8e22fe37dfd005a2c"
+}
+```
+
+=== Refreshing JWT Token
+JWT token will expire in 60 minutes. If you code requires session longer than 60 minutes consider recreating a JWT token or using refresh token.
+
+An example of how it can be done in nodejs:
```

Comment:
Similar to above, I'd just remove this code sample and explain that they may need to renegotiate the token. This is too coupled to js and to the lib we're using for http requests.